cmd/addon/addon.go

```diff
@@ -20,45 +20,31 @@ type configurator interface {
 }
 type Addon struct {
 	Name          string                 `json:"name"`
-	Repository    string                 `json:"repository,omitempty"`
-	Commit        string                 `json:"commit,omitempty"`
-	Image         string                 `json:"dockerImage,omitempty"`
-	Tag           string                 `json:"dockerTag,omitempty"`
-	Enable        bool                   `json:"enable"`
-	Configuration map[string]interface{} `json:"configuration,omitempty"`
+	Configuration map[string]interface{} `json:"configuration"`
```

Comment:
are all the removed properties optional? for example, is enabled not common to every addon? and if so should it be here?